libs/datamodel/core/src/transform/ast_to_dml/db/names.rs

```diff
@@ -28,9 +28,18 @@ pub(super) struct Names<'ast> {
     /// Datasources have their own namespace.
     pub(super) datasources: HashMap<&'ast str, TopId>,
     pub(super) model_fields: BTreeMap<(ast::ModelId, &'ast str), FieldId>,
+    pub(super) mapped_names: HashMap<MappedName, &'ast str>,
```

Comment:
Not too convinced about this one, but it's a private implementation detail we can easily change later.

Reply:
I'd say methods, methods, methods. Setters/getters whatnot. I needed to grep the project to understand where and how you fill the collections.